framework/encode/vulkan_handle_wrapper_util.h

```diff
@@ -200,9 +200,12 @@ void CreateWrappedDispatchHandle(typename ParentWrapper::HandleType parent,
         }
         if (!state_handle_table_.InsertWrapper(wrapper))
         {
-            GFXRECON_LOG_WARNING("Create a duplicated Handle: %" PRIu64
-                                 ". This wrapper can't be written into VulkanStateHandleTable.",
-                                 *handle);
+            auto existing_wrapper = state_handle_table_.GetWrapper<Wrapper>(wrapper->handle);
+            GFXRECON_LOG_WARNING("Cannot add duplicate entry to VulkanStateHandleTable for handle 0x%" PRIx64
+                                 " with ID %d. This handle is already wrapped with ID %d.",
```

Comment:
Better used `PRIu64` instead of `%d` since `handle_id` is `uint64_t`. Same for similar warning in line 226

Reply:
Fixed